webviz_subsurface/plugins/_horizon_uncertainty_viewer/horizon_uncertainty_viewer.py

```diff
@@ -29,6 +30,10 @@
 from ._huv_table import FilterTable
 
 
+@deprecated_plugin(
+    "This plugin will soon be deprecated. "
```

Comment:
```suggestion
    "The HorizonUncertaintyViewer plugin is deprecated. "
```
It is already (from now) deprecated? Similar for the other two deprecations.